backend/src/test/java/heartbeat/service/jira/JiraBoardConfigDTOFixture.java

```diff
@@ -25,7 +25,9 @@ public class JiraBoardConfigDTOFixture {
 
 	public static final String BOARD_ID = "123";
 
-	public static final String BOARD_NAME = "jira";
+	public static final String BOARD_NAME_JIRA = "jira";
```

Comment:
Can we use existing variable in main code

Reply:
Do you mean the name of BoardType?

```
JIRA("jira")
```